pipenv/environments.py

```diff
@@ -20,7 +20,7 @@
 
 # User-configuraable max-depth for Pipfile searching.
 # Note: +1 because of a temporary bug in Pipenv.
-PIPENV_MAX_DEPTH = int(os.environ.get('PIPENV_MAX_DEPTH', '3')) + 1
+PIPENV_MAX_DEPTH = int(os.environ.get('PIPENV_MAX_DEPTH', 3)) + 1
```

Comment:
The default type of `os.environ` values should be `str` and we're always casting the value to `int`. I don't think there's any particular advantage to having varying types here.